SPECS/liblogging/liblogging.spec

```diff
@@ -0,0 +1,78 @@
+Summary:        Logging Libraries
+Name:           liblogging
+Version:        1.0.6
+Release:        3%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            http://www.liblogging.org/
+Source0:        http://download.rsyslog.com/%{name}/%{name}-%{version}.tar.gz
+BuildRequires:	gcc
+
+%description
+liblogging (the upstream project) is a collection of several components.
+Namely: stdlog, journalemu, rfc3195.
+The stdlog component of liblogging can be viewed as an enhanced version of the
+syslog(3) API. It retains the easy semantics, but makes the API more
+sophisticated "behind the scenes" with better support for multiple threads
+and flexibility for different log destinations (e.g. syslog and systemd
+journal).
+
+%package devel
+Summary:        Development libraries and header files for liblogging
+Requires:       liblogging
+
+%description devel
+The package contains libraries and header files for
+developing applications that use liblogging.
+
+%prep
+%setup -q
+
+%build
+%configure --disable-journal
+make %{?_smp_mflags}
+
+%install
+make DESTDIR=%{buildroot} install
+find %{buildroot} -type f -name "*.la" -delete -print
+
+%check
+make %{?_smp_mflags} check
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%defattr(-,root,root)
+%license COPYING
+%{_bindir}/*
+%{_libdir}/*.so.*
+%{_mandir}/man1/*
+%{_mandir}/man3/*
+%{_libdir}/*.a
+
+%files devel
+%defattr(-,root,root)
+%{_libdir}/*.so
+%{_libdir}/pkgconfig/*.pc
+%{_includedir}/liblogging/*.h
+
+%changelog
+* Mon Oct 12 2020 Thomas Crain <thcrain@microsoft.com> - 1.0.6-3
+- Remove .la files
+- Lint to Mariner style
+- License verified, %license added
```

Comment:
Please use `%%` in comments. Otherwise we get warnings that a macro gets expanded in the comments or something like that.

Come to think of it, it might be worth adding to the linter's config. Same for replacing tabs with spaces.